seqr/views/apis/auth_api.py

```diff
@@ -46,6 +46,6 @@ def login_required_error(request):
 
     This is used to redirect AJAX HTTP handlers to the login page.
     """
-    assert not request.user.is_authenticated()
+    assert not request.user.is_authenticated
```

Comment:
The `is_authenticated` attribute has changed from a function to a variable in the new version of the WSGI request.